apps/web/src/app/_components/onboarding/post-onboarding/welcome-dialog.tsx

```diff
@@ -0,0 +1,30 @@
+import Image from "next/image";
+
+import { Button } from "@cooper/ui/button";
+
+interface WelcomeDialogProps {
+  heading: string;
+  subheading: string;
+  buttonText: string;
+  onClick: () => void;
+}
+
+export function WelcomeDialog({
+  heading,
+  subheading,
+  buttonText,
+  onClick,
+}: WelcomeDialogProps) {
+  return (
+    <div className="flex flex-col items-center space-y-4 text-center">
+      <Image src="/logo.png" width={150} height={150} alt="Cooper Logo" />
```

Comment:
so this is a crazy nitpick but thoughts on using svg instead bc it is giving blurry

Reply:
yes switched out the png for an svg (i took a screenshot of the one on figma instead of exporting 💀)